src/content/docs/logs/logs-context/configure-logs-context-nodejs.mdx

```diff
@@ -20,13 +20,13 @@ redirects:
   - /docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-nodejs
 ---
 
-With our Node.js APM agent, you can get **logs in context**, which lets you see your app logs in the context of your other New Relic data. For general information on this feature, see [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
+With our Node.js APM agent, you can get APM logs in context, which lets you see your app logs in the context of your other New Relic data. For general information on this feature, see [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
 
 ## Automatic logs in context options [#automatic]
 
 With application logging, your APM agent automatically collects and contextualizes your logs.
 
-If you are using a supported framework, you have three options to configure APM logs in context to send your app's logs and linking metadata automatically to New Relic.
+If you're using a supported framework, you have three options to configure APM logs in context to send your app's logs and linking metadata automatically to New Relic.
```

Comment:
```suggestion
If you're using a supported framework, you have three options to configure APM logs in context to send your app's logs and link metadata automatically to New Relic.
```

Reply:
this is referring to metadata that links New Relic with their application/hosts, so in this case it is accurately "linking metadata"